src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -319,7 +320,13 @@ def load_pytorch_state_dict_in_tf2_model(
                     continue
             raise AttributeError(f"{name} not found in PyTorch model")
 
-        array = apply_transpose(transpose, pt_state_dict[name], symbolic_weight.shape)
+        try:
+            array = apply_transpose(transpose, pt_state_dict[name], symbolic_weight.shape)
+        except tf.errors.InvalidArgumentError:
+            if not ignore_mismatched_sizes:
+                raise
```

Comment:
Here we could add `n\tYou may consider adding `ignore_mismatched_sizes=True` in the model `from_pretrained` method.` to the error message as we do in PyToch.

Reply:
Added it.